runner/src/server/views/partials/summary-row.html

```diff
@@ -11,6 +11,23 @@
                 {% endfor %}
                 {% elif item.type == 'NumberField' %}
                 {{item.prefix}}{{item.value}}
+                {% elif item.type == 'FreeTextField' %}
+                <div id={{item.name}}></div>
+                <script>
+                    // Nunjucks automatically converts any html to escaped html to prevent cross-site scripting
+                    // This method bellow safely unescapes it so it can be rendered correctly.
+                    document.getElementById('{{item.name}}').innerHTML = unescapeHtml('{{item.value}}');
+
+                    function unescapeHtml(html) {
+                        var div = document.createElement('div');
+                        div.innerHTML = html;
+                        return div.textContent;
+                      }
+                </script>
```

Comment:
Hmm.  I think you could do this with `| safe` and it would be the same - that being said, does this still open up the possibility of XSS attacks?  I mean, the applicant will have to create the attack and it'll probably only be executed on themselves... but just wondering if it's possible, might be worth trying a simple `<script>alert(1)</script>` in the box?

Reply:
So unfortunately `safe` unesacpes quotation marks as well which which leads to errors in the html. For example
Say i have a html sting 
`'<p> I can\'t render this<\p> `,

safe will then turn is into 
'<p> i can' t render this <\p>'.

So when it sets the html, its only reading the string as 
`'<p> i can'`
then it tries to read
`t render this <\p>'` and doesn't recognise t and a valid element

Also this is method doesn't run and scripts (tested this). docs on nunjucks talk about vulnerability with safe so this sort of killed two birds with one stone.

Does this make sense?